tests/protractor/e2e/tasks.js

```diff
@@ -1,137 +0,0 @@
-var utils = require('../utils');
```

Comment:
From the commit message:

> Removed tasks test which is beyond saving

I couldn't get it to work. I think it's because we log in as admin and admins don't see tasks any more, but I think it's deeper than that. In a future revision I'd like to start logging in as a restricted user instead of admin.